eth_crowdfund_be/eth_crowdfund_api/manage.py

```diff
@@ -0,0 +1,18 @@
+import os 
+from flask_script import Manager 
+from flask_migrate import Migrate, MigrateCommand 
+from src.models import Campaign, Request
+
+from src.app import create_app, db 
+
+env_name = os.getenv('FLASK_ENV')
+app = create_app(env_name)
+
+migrate = Migrate(app=app, db=db)
+
+manager = Manager(app=app)
+
+manager.add_command('db', MigrateCommand)
```

Comment:
Adds MigrateCommands (init, migrate, upgrade, etc...) to our script manager so we can run commands from the terminal